contents/handbook/growth/marketing/index.md

```diff
@@ -9,17 +9,27 @@ showTitle: true
 Philosophy: Be kind, concise and direct.
 
 
+## Positioning statement
+
+We use a positioning statement based on a template by Geoffrey Moore, which enables us to establish a position relative to the rest of the market. Our positioning statement helps us think about how we can continue to improve and evolve PostHog ahead of alternative tools.
+
+Our positioning statement is:
+
+_For product managers and software engineers who need to understand their users, PostHog is the only open source analytics platform which enables you to discover insights and run experiments._ 
+
+_Unlike traditional analytics tools, PostHog offers self-hosted deployments so that data doesn't need to be shared with anyone -- not even PostHog._ 
+
 ## Product positioning
 
-An **open source product analytics platform** addresses the lack of choice and control amongst disconnected analytics solutions by offering a **unified platform** with **control** over hosting, pricing, source, data, privacy and security.
+An **open source product analytics platform**, PostHog addresses the lack of choice and control amongst disconnected analytics solutions by offering a **unified platform** with **control** over hosting, pricing, source, data, privacy and security.
 
-**Free PostHog** is positioned to solve product analytics problems for small non-enterprise teams.
+**PostHog Open Source** is positioned to service hobbyists and very small non-enterprise teams who may want to retain control of their data. 
 
-**Enterprise PostHog** is positioned to solve product analytics problems for larger teams and enterprises.
+**PostHog Free** is positioned to service small non-enterprise teams who need to retain control of their data.
 
-**PostHog Cloud** is positioned to service clients who need less control.
+**PostHog Scale** is positioned to service large teams and enterprises who need to retain control of their data.
```

Comment:
As per the comment above, PostHog Scale is designed to service teams rather than individuals in scale ups or enterprises that need to collaborate on building a more successful product, without losing control of their data.